docs/basics.rst

```diff
@@ -0,0 +1,21 @@
+Basics
+======
+
+This section describes useful concepts across all JAXopt.
+
+Pytrees
+-------
+
+Pytrees are an essential concept in JAX and JAXopt. They can be thought as a
+generalization of vector. They are a way to structure parameters or weights
+using tuples and dictionaries. Many solvers in JAXopt have native support for
+pytrees.
+
+Double precision
+----------------
+
+JAX uses 32-bit floats by default. Howver, for some algorithms, this is not
+enough. Double precision can be enabled by adding the following at the
```

Comment:
I would add (64 bits) after precision since before precision was started in bits